src/init.ts

```diff
@@ -0,0 +1,85 @@
+import { JaaS_DOMAIN } from "./constants";
+
+type ExternalApiMap = {
```

Comment:
could you please explain the reasoning behind this? Do we expect multiple domains to be accessed by users during the lifecycle of an app?

Reply:
Yes, that was angle. But after all the feedback, it doesn't seem to ever be the case, so I'll change this part.